ios/TrueSheetView.swift

```diff
@@ -137,7 +137,7 @@ class TrueSheetView: UIView, RCTInvalidating, TrueSheetViewControllerDelegate {
     guard let containerView else { return }
 
     let size = CGSize(width: width, height: containerView.bounds.height)
-    bridge.uiManager.setSize(size, for: containerView)
+    bridge?.uiManager?.setSize(size, for: containerView)
```

Comment:
You can omit `.uiManager?` optional here

Reply:
Done